libraries/AP_RCProtocol/AP_CRSF_Out.cpp

```diff
@@ -0,0 +1,388 @@
+/*
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * AP_CRSF_Out.cpp - High-level driver for CRSF RC Output
+ *
+ * This class provides the high-level "application" logic for the
+ * CRSF RC Output feature. It is responsible for reading servo output values
+ * from the main SRV_Channels and telling its underlying CRSF protocol instance
+ * to send them at a user-configurable rate.
+ */
+#include <AP_RCProtocol/AP_RCProtocol_config.h>
```

Comment:
Not sure if this belongs in here or like AP_SBusOut as a separate library.

We only do input in this directory ATM.